PreMailer.Net/PreMailer.Net/PreMailer.cs

```diff
@@ -184,14 +174,7 @@ private IEnumerable<string> GetCssBlocks(IEnumerable<ICssSource> cssSources)
 
 			foreach (var styleSource in cssSources)
 			{
-                try
-                {
-                    styleBlocks.Add(styleSource.GetCss());
-                }
-                catch (System.Net.WebException e)
-                {
-                    _warnings.Add(e.Message);
-                }
+				styleBlocks.Add(styleSource.GetCss());

```

Comment:
Will exceptions be handled here gracefully? I wonder if we still need try-catch block?